### PR TITLE
fix the display of some known assets

### LINKED ISSUE #1172: auction ui: unknown assets errors
Unknown assets should be properly handled and filtered out, otherwise it can throw an error in the extension.

https://github.com/penumbra-zone/web/assets/70081547/3b8cc09a-7695-4705-b615-0eb64f8a420b

Screenshot 2024-05-25 at 10 43 16 AM

@jessepinho

### PR DESCRIPTION
**Pull request by Valentine1898**
close #1172

 This PR also contains other asset display fixes

### CODE REVIEW

File: apps/minifront/src/components/swap/helpers.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ const nonSwappableAssetPatterns = [
   assetPatterns.proposalNft,
   assetPatterns.votingReceipt,
   assetPatterns.auctionNft,
+  assetPatterns.lpNft,
```
Comment (Valentine1898) [inline apps/minifront/src/components/swap/helpers.ts:27]: filter out lpnft them out of the asset selectors

File: packages/ui/components/ui/tx/view/value/index.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const ValueViewComponent = ({
     const { amount = new Amount(), metadata } = view.valueView.value;
     const exponent = getDisplayDenomExponent.optional()(metadata);
     const formattedAmount = formatAmount(amount, exponent);
-    const symbol = metadata.symbol || 'Unknown Asset';
+    const symbol = metadata.symbol || metadata.display;
```
Comment (Valentine1898) [inline packages/ui/components/ui/tx/view/value/index.tsx:31]: We mistakenly displayed assets with undefined symbol as “Unknown assets”
In fact, these assets are known, and this was causing confusion. 

All  IBC assets should have a `symbol` field defined, it is required in cosmos registry 
https://github.com/cosmos/chain-registry/blob/62b3fa019a0db5a277925859999ab430420a3fa6/assetlist.schema.json#L36

But some penumbra internal assets (such as lpnft) don't have this field, so we have to use `display`

**Comment by TalDerei** [comment]
filtered UI looks much cleaner!

**Review by TalDerei — APPROVED** [review]

File: apps/minifront/src/components/shared/asset-selector.tsx [hunk 3]
```diff
@@ -127,7 +127,7 @@ export const AssetSelector = ({ assets, onChange, value, filter }: AssetSelector
                     onClick={() => onChange(metadata)}
                   >
                     <AssetIcon metadata={metadata} />
-                    <p className='truncate'>{metadata.symbol}</p>
+                    <p className='truncate'>{metadata.symbol || metadata.display}</p>
```
[the remark this answers is not in the document]
Comment (grod220) [reply apps/minifront/src/components/shared/asset-selector.tsx:130]: Yeah, we don't want to show the display here, looks like you address this in https://github.com/penumbra-zone/web/pull/1213

**Pull request merged**